themes/SuiteP/css/dashboardstyle.css

```diff
@@ -54,7 +54,7 @@
     float: left;
     margin-top: 6px;
     
-    color: #534d64;
+    color: #196579;
```

Comment:
Updating commented code :O. These should be removed if we don't need them anymore.